src/openforms/forms/models/form_version.py

```diff
@@ -0,0 +1,34 @@
+import uuid as _uuid
+
+from django.contrib.postgres.fields.jsonb import JSONField
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from openforms.utils.fields import StringUUIDField
+
+
+class FormVersion(models.Model):
+    uuid = StringUUIDField(_("UUID"), unique=True, default=_uuid.uuid4)
+
+    date_creation = models.DateTimeField(
```

Comment:
using `date` in a field name that's actually a datetime always makes me feel a bit uneasy as the name gives the wrong impression.

```suggestion
    created = models.DateTimeField(
```